eth/tracers/native/call.go

```diff
@@ -35,16 +36,17 @@ func init() {
 }
 
 type callFrame struct {
-	Type    string      `json:"type"`
-	From    string      `json:"from"`
-	To      string      `json:"to,omitempty"`
-	Value   string      `json:"value,omitempty"`
-	Gas     string      `json:"gas"`
-	GasUsed string      `json:"gasUsed"`
-	Input   string      `json:"input"`
-	Output  string      `json:"output,omitempty"`
-	Error   string      `json:"error,omitempty"`
-	Calls   []callFrame `json:"calls,omitempty"`
+	Type         string      `json:"type"`
+	From         string      `json:"from"`
+	To           string      `json:"to,omitempty"`
+	Value        string      `json:"value,omitempty"`
+	Gas          string      `json:"gas"`
+	GasUsed      string      `json:"gasUsed"`
+	Input        string      `json:"input"`
+	Output       string      `json:"output,omitempty"`
+	Error        string      `json:"error,omitempty"`
+	RevertReason string      `json:"revertReason,omitempty"`
```

Comment:
Perhaps call it `Revertal`, the current `ReverReason` look unwieldy among the short names.